libr/include/r_anal.h

```diff
@@ -2052,6 +2059,7 @@ R_API void r_anal_class_list(RAnal *anal, int mode);
 R_API void r_anal_class_list_bases(RAnal *anal, const char *class_name);
 R_API void r_anal_class_list_vtables(RAnal *anal, const char *class_name);
 R_API void r_anal_class_list_vtable_offset_functions(RAnal *anal, const char *class_name, ut64 offset);
+R_API RGraph *r_anal_class_get_inheritance_graph(RAnal *anal);
```

Comment:
The type of user data inside RGraph is part of interface which needs to be documented the same way it is for RList or RVector.